### PR TITLE
chore: improve doc about azure devops

### PR DESCRIPTION
**Pull request by krol3**

### CODE REVIEW

**Comment by krol3** [comment]
@liamg @AnaisUrlichs  please review this doc for the integration with Azure devops

**Review by liamg — APPROVED** [review]
:+1:

**Review by knqyf263 — APPROVED** [review]

**Comment by knqyf263** [comment]
I accidentally approved it, but we probably need to update mkdocs.yml as well.

File: docs/docs/integrations/azure-devops.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Azure Devops
+
+- Here is the [Azure DevOps Pipelines Task for Trivy][action]
+
+![trivy-azure](https://github.com/aquasecurity/trivy-azure-pipelines-task/blob/main/screenshot.png?raw=true)
+
+### [Microsoft Defender for container registries and Trivy][azure]
+
+The findings of the CI/CD scans are an enrichment to the existing registry scan findings by Qualys. Defender for Cloud's CI/CD scanning is powered by Aqua Trivy
```
Comment (AnaisUrlichs) [inline docs/docs/integrations/azure-devops.md:9]: Could we add a bit more information -- not knowing what it is, it's a bit difficult to understand why I should look into it e.g. as an Azure user

**Comment by AnaisUrlichs** [comment]
Added a comment 👍🏼

File: docs/docs/integrations/azure-devops.md (hunk 1 above)
Comment (krol3) [reply docs/docs/integrations/azure-devops.md:13]: @AnaisUrlichs  I added more context from the azure blog

**Comment by krol3** [comment]
> I accidentally approved it, but we probably need to update mkdocs.yml as well.

I updated the mkdocs @knqyf263

File: docs/docs/integrations/azure-devops.md (hunk 1 above)
Comment (AnaisUrlichs) [reply docs/docs/integrations/azure-devops.md:13]: Perfect 😄 thank you

**Review by knqyf263 — APPROVED** [review]

**Comment by knqyf263** [comment]
Thanks!

**Pull request merged**